src/components/CharacterInfo.vue

```diff
@@ -11,8 +11,31 @@ import CharacterInfoPortrait from "./CharacterInfoPortrait.vue";
  */
 export default {
   components: [CharacterInfoInput, CharacterInfoSelect, CharacterInfoPortrait],
-  props: ["bio", "generations"],
+  props: ["bio", "predatorDefinitions"],
   emits: ["bioChange"],
+  computed: {
+    /**
+     * @returns list of disciplines filtered by predator type value from predatorDefinitions
+     */
+    predatorTypeDisciplines() {
+      return this.predatorDefinitions.filter(
+        (predator) => predator.id == this.bio.predator.value
+      )[0].disciplines
+    }
+  },
+  methods: {
+    /**
+     * after change of predator type check if we have to remove predator discipline
+     */
+    chosenDisciplineCheck(predatorType) {
+      var allowedDisciplines = this.predatorDefinitions.filter(
+        (predator) => predator.id == predatorType
+      )[0].disciplines
+      if (! allowedDisciplines.includes(this.bio.predator.chosenDiscipline)) {
+        this.bio.predator.chosenDiscipline = "no"
```

Comment:
alebo ze by mi vadilo ze tu sa mutuje nejaka prop?

Reply:
to mas pravdu, upravil som. Diky, ze si to vytiahol